### DISCUSSION TITLE
Combining game html5

### DISCUSSION THREAD

**Discussion by Nega100** [jspsych/jsPsych#2790] [General]
Hi all
I have trouble combining game construct into jspsych. is this possible to create construct game inside jspysch plugin?. 

Thanks

**Comment by ChristopheBossens** [comment]
The plugin system is very flexible so in principle you can combine anything within it. Can you try to explain in more detail what you are trying to do specifically, and maybe show us some of the things you have tried already and why those things are not working?

**Reply by Nega100** [reply]
Thanks for the answer. The construct code seems encrypted so I cant move in jspsych plugin. Is it possible to modifiy main.js fully understandable like phaser.js.

**Reply by ChristopheBossens** [reply]
Still not sure what you are trying to achieve. Could you maybe try to explain what the purpose of the experiment is that you are trying to set up?

Since you mentioned phaser.js, that's already a framework by itself. I was just thinking it might be easier to code your whole experiment in phaser.js? Or what is the exact reason you want to combine both phaser and jsPsych?
